### DISCUSSION TITLE
If there are many nodes and edges, the reaflow may cause the browser to break. How to use the reaflow to display the flow chart when there are many nodes and edges?

### DISCUSSION THREAD

**Discussion by niuzhi** [reaviz/reaflow#255] [Q&A]
If there are many nodes and edges, the reaflow may cause the browser to break. How to use the reaflow to display the flow chart when there are many nodes and edges?

**Comment by Ononokierii** [comment]
I have also encountered this situation. When I create more than 100 nodes and classify them by connecting with each other, the browser crashes. I want to try using virtual scrolling to render, but I'm not sure if it's feasible.
